HotelPriceScout/Pages/PriceMeterGenerator.cs

```diff
@@ -7,23 +7,24 @@ namespace HotelPriceScout.Pages
 {
         public class PriceMeterGenerator
         {
-            public static List<Prices> PriceListGenerator(DateTime TodayDate, IEnumerable<MarketPriceModel> MonthData) // generates a test list of hotels and their prices.
+            public static List<MarketPriceModel> PriceListGenerator(DateTime TodayDate, IEnumerable<MarketPriceModel> MonthData) // generates a test list of hotels and their prices.
             {
-                List<Prices> PriceDataList = new();
+                List<MarketPriceModel> PriceDataList = new();
                 PriceDataList.AddRange(from MarketPriceModel item in MonthData
```

Comment:
Det ser ud som om at der stod MarketPriceModel i forvejen her i stedet for Prices. Vil dette have nogen indflydelse?